src/mixins/Mime.js

```diff
@@ -50,12 +50,12 @@ export default {
 		// file path relative to user folder
 		hasPreview: {
 			type: Boolean,
-			required: true,
+			default: false,
 		},
 		// unique file id
 		fileid: {
 			type: [Number, String],
-			required: true,
+			required: false,
```

Comment:
Does this work ? I though `required` defaulted to `false` but in this case we had to set `default`.